src/applications/vaos/utils/error.js

```diff
@@ -0,0 +1,13 @@
+import * as Sentry from '@sentry/browser';
+
+export function captureError(err) {
```

Comment:
This should not be necessary, but the `apiRequest` helper is returning an object as the data for a failed promise, when Error objects are recommended. So we need to make sure we use the right Sentry call for the right type of error data.

Reply:
Nice catch.  I wasn't aware of this.